<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,0">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Setting file</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0">
               <item>
                <widget class="QLineEdit" name="setting_fileEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="open_dialogButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>70</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="save_fileButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>70</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTextEdit" name="setting_textEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Event settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>170</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Event</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <widget class="QRadioButton" name="both_radioButton">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Both</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="on_change_radioButton">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>On change</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="timer_radioButton">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Timer</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="verticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_5">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>170</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Timer properties</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout">
                      <item>
                       <widget class="QDoubleSpinBox" name="time_spinBox">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>8</pointsize>
                         </font>
                        </property>
                        <property name="decimals">
                         <number>3</number>
                        </property>
                        <property name="minimum">
                         <double>0.001000000000000</double>
                        </property>
                        <property name="maximum">
                         <double>99999.990000000005239</double>
                        </property>
                        <property name="singleStep">
                         <double>0.100000000000000</double>
                        </property>
                        <property name="value">
                         <double>1.000000000000000</double>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>second(s)</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_3">
                      <item>
                       <spacer name="horizontalSpacer">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QPushButton" name="save_timeButton">
                        <property name="minimumSize">
                         <size>
                          <width>50</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>50</width>
                          <height>25</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>OK</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <spacer name="verticalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="title">
                <string>Tags for monitoring</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QListWidget" name="tags_listWidget">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="lineWidth">
                   <number>0</number>
                  </property>
                  <property name="tabKeyNavigation">
                   <bool>false</bool>
                  </property>
                  <property name="showDropIndicator" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="dragEnabled">
                   <bool>false</bool>
                  </property>
                  <property name="dragDropOverwriteMode">
                   <bool>false</bool>
                  </property>
                  <property name="dragDropMode">
                   <enum>QAbstractItemView::DragDrop</enum>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>false</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectItems</enum>
                  </property>
                  <property name="textElideMode">
                   <enum>Qt::ElideLeft</enum>
                  </property>
                  <property name="flow">
                   <enum>QListView::TopToBottom</enum>
                  </property>
                  <property name="resizeMode">
                   <enum>QListView::Fixed</enum>
                  </property>
                  <property name="layoutMode">
                   <enum>QListView::SinglePass</enum>
                  </property>
                  <property name="viewMode">
                   <enum>QListView::IconMode</enum>
                  </property>
                  <property name="uniformItemSizes">
                   <bool>false</bool>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                  <property name="selectionRectVisible">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="open_tags_window">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Tags</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mainwindow">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>MainWindow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectcamera">
          <property name="text">
           <string>select camera</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
